extensions/wikia/TrackingOptIn/js/cmp.js

```diff
@@ -227,6 +266,25 @@ define('wikia.cmp', [
 				}, {})
 			}, true);
 		},
+		getPublisherConsent: function (purposesIds, callback) {
+			var purposesAllowed = (purposesIds || publisherConsent.getPurposesAllowed()).reduce(function (obj, id) {
+					var type = (id > 24) ? 'customPurposeConsents' : 'standardPurposeConsents';
```

Comment:
What's the magic `24`? Can we assign it to a variable with a descriptive name? ;-)

Reply:
@nandy-andy id's up to 24 are reserved. 25+ may be our custom purposes :)